src/spaceone/inventory/service/cloud_service_service.py

```diff
@@ -181,30 +176,42 @@ def update(self, params):
         old_cloud_svc_data = dict(cloud_svc_vo.to_dict())
 
         if 'tags' in params:
-            new_tags = self._create_tag(params, tag_type)
+            old_tags = old_cloud_svc_data.get('tags', {})
+            new_tags = self._create_tags(params['tags'], provider)
```

Comment:
The two methods seem to perform the same action, so please combine them into one.
And the function name looks better `convert` than `create`.

Reply:
have reflected thank you.